constance/backends/database/__init__.py

```diff
@@ -87,7 +86,7 @@ def set(self, key, value):
         if self._cache:
             self._cache.set(key, value)
 
-        updated_signal.send(sender='constance', key=key, value=value)
+        signals.config_updated.send(sender='constance', key=key, value=value)
```

Comment:
Ah, good point, it's cheap for the redis backend, but for the db backend you're right